src/openms/source/IONMOBILITY/IMDataConverter.cpp

```diff
@@ -78,7 +78,7 @@ namespace OpenMS
     // fill up the PeakMaps by moving spectra from the input PeakMap
     for (const MSSpectrum& it : exp)
```

Comment:
```suggestion
    for (MSSpectrum& it : exp)
```

Reply:
will add to https://github.com/OpenMS/OpenMS/pull/5685